### ISSUE TITLE
tableTypes should create Pipes Consumer

### ISSUE THREAD

**Issue by codygman** [acowley/Frames#62]
In my workflow I constantly have:

```haskell
tableTypes "User" "data/ml-100k/u.user"
movieStream :: Producer User IO ()
movieStream = readTableOpt userParser "data/ml-100k/u.user"
```

I'd like to just shorten this to:

```haskell
tableTypes "User" "data/ml-100k/u.user"
```

And have movieStream created for me automatically. I don't typically have a reason for my table types filepath and movieStream filepath to differ, though I acknowledge this could be a valid use case.

Perhaps we could have a default of using the same filepath in both places and override it, e.g.:

default:

```haskell
tableTypes "User" "data/ml-100k/u.user"
```

override:

```haskell
tableTypes "User" "data/ml-100k/u.user" "someotherplace"
```

**Comment by acowley** [comment]
Great idea. I've felt the itch of this boilerplate, too.
